views/geneticSort.pug

```diff
@@ -0,0 +1,95 @@
+doctype html
+html
+  head
+    meta(charset='UTF-8')
+    title Topological Sorting
+  style.
+    .axis path,
```

Comment:
Better write this part in public/css. It's okay to remain this part unchanged. If you already run out of the time, I will move it by myself.

Reply:
done.